### PR TITLE
chore: clarify properties for windows e2e

### PR DESCRIPTION
**Pull request by junjiezhang1997**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
1. remove unnecessary properties (duplicate properties in nodebootstrapping_template.json and properties for linux) for windows e2e.
2. specify distro and node image version to avoid ambiguity.

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: .pipelines/e2e-windows.yaml [hunk 1]
```diff
@@ -41,6 +41,8 @@ stages:
       windowsImage: 2019-containerd
       windowsOSSKU: Windows2019
       windowsVMSize: Standard_D2s_v3
+      windowsDistro: aks-windows-2019-containerd
+      windowsNodeImageVersion: AKSWindows-2019-containerd-test-2023.02.07
```
Comment (ShiqianTao) [inline .pipelines/e2e-windows.yaml:45]: Why is the value `2023.02.07`?

Comment (junjiezhang1997) [reply .pipelines/e2e-windows.yaml:45]: Currently we use the test image referred in Line 139 of e2e/windows/e2e-scenario.sh and it was built by ourselves on 2023/02/07. 

In the future, after moving our vhd sub to ame sub, we can use prod image and sync with the latest image.

File: .pipelines/e2e-windows.yaml [hunk 2]
```diff
@@ -41,6 +41,8 @@ stages:
       windowsImage: 2019-containerd
       windowsOSSKU: Windows2019
       windowsVMSize: Standard_D2s_v3
+      windowsDistro: aks-windows-2019-containerd
```
Comment (AbelHu) [inline .pipelines/e2e-windows.yaml:44]: Can you help to add a README for how to generate and update the test images, how to generate the template, the brief introduction of windows e2e, etc.?

Comment (junjiezhang1997) [reply .pipelines/e2e-windows.yaml:44]: Could I create a new task rather than in this pr to do it? I think it may take some time.

**Review by AbelHu — APPROVED** [review]
LGTM

**Pull request merged**